.pre-commit-config.yaml

```diff
@@ -69,7 +69,13 @@ repos:
     rev: "v2.3.0"
     hooks:
       - id: codespell
-        args: ["--write-changes", "--ignore-words", ".codespell-whitelist"]
+        args:
+          [
+            "--write-changes",
+            "--ignore-words",
+            ".codespell-whitelist",
+            "--ignore-words-list=tE",
```

Comment:
You can add `tE` to the codespell-whitelist. It should be `.codespell-whitelist` in the main directory of caustics